src/Searchbar.vue

```diff
@@ -15,41 +19,66 @@ export default {
       if (!this.data) {
         return undefined;
       }
-      const matches = [];
+      const pages = [];
       const regexes = this.value.split(' ')
         .filter(searchKeyword => searchKeyword !== '')
         .map(searchKeyword => searchKeyword.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'))
-        .map(searchKeyword => new RegExp(searchKeyword, 'i'));
+        .map(searchKeyword => new RegExp(searchKeyword, 'ig'));
```

Comment:
I guess if the aim is to find the total number of matches (rather than the total number of regexes it matches), it seems reasonable to have it. Is that the expected behaviour here?